### PR TITLE
task: configure CircleCI

### PR DESCRIPTION
**Pull request by KubaBaniak**
## [Configure CI pipeline](https://app.clickup.com/t/862k1pe9j)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    docker:
+      - image: cimg/base:2023.07
+      - image: cimg/postgres:14.1
+
+    steps:
+      - checkout
+      - install_dependencies
+      - setup_prisma
+      - setup_docker_compose
+      - run: echo "BUILDING PROJECT"
```
Comment (arkus7) [inline .circleci/config.yml:68]: I guess this is a placeholder?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      - run:
+          name: Run unit tests
+          command: npm run test
+
+  deploy:
+    working_directory: ~/repo
+
+    docker:
+      - image: cimg/base:2023.07
+      - image: cimg/postgres:14.1
```
Comment (arkus7) [inline .circleci/config.yml:61]: Why do you need postgres for deployment?

**Review by arkus7 — CHANGES_REQUESTED** [review]
Use the `cimg/node` image instead of `base`

File: .circleci/config.yml (hunk 2 above)
Comment (KubaBaniak) [reply .circleci/config.yml]: Thought, I have to deploy the app as well as db for it. With that suggestion I assume db will be hosted on different server?

**Review by arkus7 — APPROVED** [review]

**Pull request merged**